Framework/PythonInterface/mantid/fitfunctions.py

```diff
@@ -0,0 +1,524 @@
+from mantid.simpleapi import FunctionFactory
```

Comment:
As the `simpleapi` imports this and this then imports the `simpleapi` it would be better if this line just read `from mantid.api import FunctionFactory`